README.md

```diff
@@ -143,14 +146,29 @@ Note, that today's date really depends on the time zone in which you're observin
 val knownDate = LocalDate(2020, 2, 21)
 ```
 
+### Getting local time components
+
+`LocalTime` type represents local time without date. You can obtain it from `Instant`
+by converting it to `LocalDateTime` and taking its `time` property.
+
+```kotlin
+val now: Instant = Clock.System.now()
+val thisTime: LocalTime = now.toLocalDateTime(TimeZone.currentSystemDefault()).time
+```
+
+`LocalTime` can be constructed from four components, hour, minute, second and nanosecond:
```

Comment:
Might be worth showing an example with just H/M/S, and even another example with just H/M?

Reply:
sure, I'll try to see what I can add